custom/icds/location_reassignment/dumper.py

```diff
@@ -37,7 +37,10 @@ def dump(self, transitions_per_location_type):
         headers = [[location_type, DUMPER_COLUMNS]
                    for location_type in location_types]
         stream = io.BytesIO()
-        self._setup_site_codes(list(transitions_per_location_type.values()))
+        all_transitions = []
+        for transitions in transitions_per_location_type.values():
```

Comment:
Update dumper to adapt to new structure where
`transitions_per_location_type` is a dict with location_type_code mapped to
a list of Transition objects, instead of the old structure.